libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -107,6 +107,11 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
       flex-shrink: 0;
       flex-grow: 0;
     }
+
+    span {
+      align-self: center;
```

Comment:
Why is this necessary? 🤔

Reply:
When I initially set `align-self: center` line-height was not set and the label was not properly center-aligned with the actual radio/checkbox:
<img width="120" alt="Screenshot 2021-03-11 at 10 09 13" src="https://user-images.githubusercontent.com/17269698/110763129-d60bee00-8251-11eb-83ae-fc6294b768cd.png">

After applying `align-self: center` the label was properly aligned: 
<img width="128" alt="Screenshot 2021-03-11 at 10 11 56" src="https://user-images.githubusercontent.com/17269698/110763469-369b2b00-8252-11eb-941d-7c4d96793871.png">

Now that you point it out I can however see that `align-self: center` is no longer necessary due to the line-height matching the radio/checkbox - so I will go ahead and remove it 😄

Reply:
Should be resolved by: a8e1e31